google/cloud/alloydb/connector/instance.py

```diff
@@ -33,6 +34,29 @@
 
 logger = logging.getLogger(name=__name__)
 
+INSTANCE_URI_REGEX = re.compile(
+    "projects/([^:]+(:[^:]+)?)/locations/([^:]+)/clusters/([^:]+)/instances/([^:]+)"
+)
+
+
+def parse_instance_uri(instance_uri: str) -> Tuple[str, str, str, str]:
```

Comment:
nit: let's add the leading underscore to mark this as an internal function that public users shouldn't import
```suggestion
def _parse_instance_uri(instance_uri: str) -> Tuple[str, str, str, str]:
```

Reply:
Hi @jackwotherspoon, I have changed the visibility of the function to private in the latest push.